rudder-core/src/main/resources/hooks.d/node-pre-deletion/10-log-node-pre-deletion

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
I'm not sure we should keep that one. If one should be kept by default, it should better be a log in post-delete place, saying "node was deleted". We could even add a real clean-up hook for node promises, for example (3 rm -rf - with a check that the path starts by /var/rudder/share, just to be sure we are not rm'ing the whole system - one for each RUDDER_POLICIES_DIRECTORY. Or perhaps keeps the archive, for post-mortem)